Makefile

```diff
@@ -2,26 +2,50 @@ HOSTNAME=github.com
 NAMESPACE=iterative
 NAME=iterative
 VERSION=0.0.${shell date +%s}+development
-OS_ARCH=${shell go env GOOS}_${shell go env GOARCH}
-BINARY=terraform-provider-${NAME}
-INSTALL_PATH=~/.terraform.d/plugins/${HOSTNAME}/${NAMESPACE}/${NAME}/${VERSION}/${OS_ARCH}
+GOOS=${shell go env GOOS}
+GOARCH=${shell go env GOARCH}
+TF_PLUGIN_INSTALL_PATH=~/.terraform.d/plugins/${HOSTNAME}/${NAMESPACE}/${NAME}/${VERSION}/${GOOS}_${GOARCH}
+TPI_PATH ?= $(shell pwd)
+LEO_PATH ?= $(shell pwd)/cmd/leo
```

Comment:
`shell pwd` won't produce the expected result when using `make --directory`; it might be simpler to just use relative paths.

Reply:
It would actually 🤔 , `make` changes the dir before doing anything else so it works as expected.
To illustrate, you can replace the current `Makefile` contents with this (used a var just to illustrate a 1:1 analog):
```make
PWDVAR = $(shell pwd)


default: printme

.PHONY: printme
printme:
	@echo ${PWDVAR}
```

Then, a simple test run inside this repo would show:
```bash
❯ make
/Users/oded/iterative/workspace/terraform-provider-iterative
❯ cd cmd
❯ make --directory ../
/Users/oded/iterative/workspace/terraform-provider-iterative
```

Hope I didn't miss anything 😄

Reply:
Major blunder on my side: `${PWD}` is the one I was thinking of, but `$(shell pwd)` or even `$(CURDIR)` behave as expected. 🤦🏼‍♂️